### ISSUE TITLE
need script for settings names for consistency

### ISSUE THREAD

**Issue by ThatsGottaBeKane** [ThatsGottaBeKane/OoTMMR_tracker_pack#13]
without it, we cannot reproduce the settings names as wbsch and jupiterfire have done we cannot finish several settings until we get this.  just need to wait on this, making an issue just so it's not forgotten

**Comment by Tsarchasm5** [comment]
no longer needed
